server/src/test/java/org/elasticsearch/index/codec/tsdb/es819/ES819TSDBDocValuesFormatTests.java

```diff
@@ -757,22 +761,27 @@ public void testOptionalColumnAtATimeReader() throws Exception {
         final String counterFieldAsString = "counter_as_string";
         final String timestampField = "@timestamp";
         final String gaugeField = "gauge";
+        final String binaryField = "binary";
```

Comment:
Maybe add two binary fields, one with fixed length and one with variable length?